Failing auth server unit test: selectEmailServices: ... selectEmailServices returns the correct data:

>  1) selectEmailServices:
       redis.get returns sendgrid percentage-only mismatch:
         redis.get returns sendgrid regex-only match:
           selectEmailServices returns the correct data:

      AssertionError: expected [ Array(1) ] to deeply equal [ Array(1) ]
      + expected - actual

         {
           "emailAddresses": [
             "foo@example.com"
           ]
      -    "emailSender": "ses"
      -    "emailService": "fxa-auth-server"
      +    "emailSender": "sendgrid"
      +    "emailService": "fxa-email-service"
           "mailer": {
      -      "mailer": true
      +      "emailService": true
           }
         }
       ]

      at Context.it (test/local/senders/select_email_services.js:94:16)

--- a/packages/fxa-auth-server/test/local/senders/email.js
+++ b/packages/fxa-auth-server/test/local/senders/email.js
@@ -845,8 +845,8 @@ describe('lib/senders/email:', () => {
     assert.notEqual(mailer.mailer, mailer.emailService);
   });
 
-  it('declarative tests', async () => {
-    for (const [type, test] of TESTS) {
+  for (const [type, test] of TESTS) {
+    it(`declarative test for ${type}`, async () => {
       mailer.mailer.sendMail = stubSendMail(message => {
         COMMON_TESTS.forEach((assertions, property) => {
           applyAssertions(type, message, property, assertions);
@@ -858,8 +858,8 @@ describe('lib/senders/email:', () => {
       });
 
       await mailer[type](MESSAGE);
-    }
-  });
+    });
+  }
 
   it('formats user-agent strings sanely', () => {
     let result = mailer._formatUserAgentInfo({

